doc/release-notes/5.10.1-release-notes.md

```diff
@@ -0,0 +1,96 @@
+# Dataverse Software 5.10.1
+
+This release brings new features, enhancements, and bug fixes to the Dataverse Software. Thank you to all of the community members who contributed code, suggestions, bug reports, and other assistance across the project.
+
+## Release Highlights
+
+### Bug Fix for Request Access
+
+Dataverse Software 5.10 contains a bug where the "Request Access" button doesn't work from the file listing on the dataset page if the dataset contains custom terms. This has been fixed in PR #8555.
+
+### Bug Fix for Searching and Selecting Controlled Vocabulary Values
+
+Dataverse Software 5.10 contains a bug where the search option is no longer present when selecting from more than ten controlled vocabulary values. This has been fixed in PR #8521
+
+## Major Use Cases and Infrastructure Enhancements
+
+Changes and fixes in this release include:
+
+- Users can use the "Request Access" button when the dataset has custom terms. (Issue #8553, PR #8555)
+- Users can search when selecting from more than ten controlled vocabulary values. (Issue #8519, PR #8521)
+- The default file categories ("Documentation", "Data", and "Code") can be redefined through the `:FileCategories` database setting. (Issue #8461, PR #8478)
+- Documentation on troubleshooting Excel ingest errors was improved. (PR #8541)
+- Internationalized controlled vocabulary values can now be searched. (Issue #8286, PR #8435)
+- Curation labels can be internationalized. (Issue #8381, PR #8466)
+- "NONE" is no longer accepted as a license using the SWORD API (since 5.10). See "Backward Incompatibilities" below for details. (Issue #8551, PR #8558).
+
+## Notes for Dataverse Installation Administrators
+
+### PostgreSQL Version 10+ Required Soon
+
+Because 5.10.1 is a bug fix release, an upgrade to PostgreSQL is not required. However, this upgrade is still coming in the next non-bug fix release. For details, please see the release notes for 5.10: https://github.com/IQSS/dataverse/releases/tag/v5.10
+
+### Payara Upgrade
+
+You may notice that Payara was updated from 5.2021.5 to 5.2021.6. This was to address a bug where it was not possible to easily update the logging level but an upgrade is not strictly necessary. For more information, see PR #8508.
```

Comment:
Some additional text to point out that this is for new releases and existing installations don't have to update payara, and that that isn't part of the upgrade instructions below might be useful. E.g.:
You may notice that the Payara version used in the install scripts has been updated from 5.2021.5 to 5.2021.6. This was to address a bug where it was not possible to easily update the logging level. For existing installations, this release does not require upgrading payara and a payara upgrade is not part of the Upgrade Instructions below. For more information, see PR #8508.